docs/topics/testing/tools.txt

```diff
@@ -138,7 +151,7 @@ Use the ``django.test.Client`` class to make requests.
     Once you have a ``Client`` instance, you can call any of the following
     methods:
 
-    .. method:: Client.get(path, data=None, follow=False, secure=False, **extra)
+    .. method:: Client.get(path, data=None, follow=False, secure=False, *, headers=None, **extra)
```

Comment:
`Client` has its own implementation of `get()`, `post()`, `head()`, `options()`, `put()`, `patch()`, `delete()`, and `trace()` methods so signatures didn't change. As far as I'm aware, we should add the `headers` kwarg to the `Client`'s methods.

Reply:
Wouldn't it make sense to change the `Client` methods to add the `*` and `headers` parameter? 🤔 
I guess it would provide a more explicit interface for users of `Client`.

Reply:
> Wouldn't it make sense to change the Client methods to add the * and headers parameter?

Yes, that's what I was trying to ask for :smile: